### ISSUE TITLE
[STY Go-Live] Enable batch#1 smoke tests on live

### ISSUE THREAD

**Issue by chris-hinds** [bbc/simorgh#6599]
**Is your feature request related to a problem? Please describe.**
With Batch#1 of STY's going live soon we should ensure the services in the batch have Mozart smoke test enabled on Live.

Blocked on #6598 
**DO NOT MERGE** until #6598 has been merged and is live.

**Services in Batch#1**
- ukchina
- gahuza

**Describe the solution you'd like**
Enable live Mozart smoke test for the services in batch#1

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

**Comment by chris-hinds** [comment]
Closing a duplicate of https://github.com/bbc/simorgh/issues/6349
